{"name": "Невозможность изменения данных неавторизованного пользователя, email", "status": "passed", "description": "При попытке изменения {field} возвращает HTTP 401 и сообщение об ошибке", "steps": [{"name": "Отправка POST на auth/login", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'dtran@example.net', 'password': '6218531466', 'name': 'John Howard'}"}], "start": 1718207480263, "stop": 1718207480551}, {"name": "Отправка PATCH на auth/user", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'abracadabra', 'password': '6218531466', 'name': 'John Howard'}"}, {"name": "access_token", "value": "''"}], "start": 1718207480551, "stop": 1718207480766}], "parameters": [{"name": "field", "value": "'email'"}], "start": 1718207480263, "stop": 1718207480767, "uuid": "a275bcda-6ab3-4890-9ee3-81f075f22f76", "historyId": "00b4f347846d088a245d73b592509f8b", "testCaseId": "f450658db434b016c441fc12e73231b2", "fullName": "tests.test_auth_user.TestChangeUserData#test_change_user_data_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "XPMEPODW0067"}, {"name": "thread", "value": "9424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}